src/App.tsx

```diff
@@ -30,6 +31,10 @@ import ExploreScreen from '@nightlight/screens/explore/ExploreScreen';
 import SocialScreen from '@nightlight/screens/social/SocialScreen';
 import ProfileScreen from '@nightlight/screens/profile/ProfileScreen';
 import EmergencyContactsScreen from '@nightlight/screens/profile/EmergencyContactsScreen';
+import * as Device from 'expo-device';
+import * as Notifications from 'expo-notifications';
```

Comment:
I personally don't use wildcard unless it is used for exports (i.e. `export * as Device from '...';`)

Otherwise, we should selectively import the parts we need rather than polluting the space.